frontend/src/pages/node/AssociationsTable.vue

```diff
@@ -332,6 +327,29 @@ async function download() {
   );
 }
 
+const frequencyPercentage = (row: DirectionalAssociation) => {
+  // Returns decimal, convert to percentage in template
+  if (row.has_percentage) return row.has_percentage / 100;
+  else if (row.has_count && row.has_total) return row.has_count / row.has_total;
+  else if (row.frequency_qualifier) {
+    switch (row.frequency_qualifier) {
+      case "HP:0040280":
+        return 1.0;
```

Comment:
Decimal points on integers aren't necessary because all numbers in js are floats anyway.